docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -50,5 +50,48 @@ classpath:/jbang.properties
    wrapper.install.dir = .
 ```
 
+== Proxy configuration
 
+If you use proxies there are a few settings to apply to make JBang *fully* work with proxies.
+
+First the proxy settings for bash/zsh scripting part (needed if jbang nor java is not already installed). This is done by setting the `http_proxy` and `https_proxy` environment variables. For example:
+
+```bash
+http_proxy=http://proxyhost:8888
```

Comment:
I wonder if we need to explain what to do on Windows. But I'm not sure how to test. I have Windows, but not sure how to simulate a proxy.

Reply:
Good point - i reckon on Windows use system proxies actually Works.

Reply:
I've updated with some more system context.